media-video/celluloid/celluloid-0.29.ebuild

```diff
@@ -0,0 +1,36 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit gnome2-utils meson xdg
+
+DESCRIPTION="Simple GTK+ frontend for mpv"
+HOMEPAGE="https://celluloid-player.github.io/"
+SRC_URI="https://github.com/celluloid-player/celluloid/releases/download/v${PV}/${P}.tar.xz"
+
+LICENSE="GPL-3+"
+SLOT="0"
+KEYWORDS="~amd64 ~arm64 ~ppc64"
```

Comment:
Is there a reason for removing ~ppc ?
Did you skip https://github.com/gentoo/gentoo/commit/06b8bf5af1366a701db2f244ebcecb05dfc22f9e

Reply:
@jospezial , yes, I did.
Fixed.